nvm.sh

```diff
@@ -2317,9 +2317,14 @@ nvm() {
       NVM_IOJS_PREFIX="$(nvm_iojs_prefix)"
       local NVM_NODE_PREFIX
       NVM_NODE_PREFIX="$(nvm_node_prefix)"
+      local GREENVERSION
+      GREENVERSION='\033[0;32m%s\033[0m'
```

Comment:
Colorization should only happen if `nvm_has_colors` returns true.

Reply:
I just add a condition as you ve said

```sh
      local NVM_NO_COLOR
      local GREENVERSION
      if [ -z "${NVM_NO_COLORS-}" ] && nvm_has_colors; then
        NVM_HAS_COLORS=1
      fi
      NVM_VERSION="$(nvm --version)"
      if [ "${NVM_HAS_COLORS-}" = '1' ]; then
        GREENVERSION='\033[0;32m%s\033[0m\n'
      else
        GREENVERSION=' %12s *'
      fi
```
i pushed it in new commit